Gruntfile.js

```diff
@@ -576,6 +614,14 @@ module.exports = function(grunt) {
         'build-assets',
         'package-report',
     ]);
+    grunt.registerTask('build-package-ui', [
+        'clean:intermediates',
+        'exec:generate-scss-typings',
+        // 'exec:webpack-prod', // required to get the css assets
```

Comment:
should we remove this commented line here?

Reply:
I guess so. :)

Reply:
Removed